tests/projects/test_databricks.py

```diff
@@ -266,40 +269,16 @@ def test_run_databricks_cluster_spec_json(
         runs_submit_args, _ = runs_submit_mock.call_args_list[0]
         req_body = runs_submit_args[0]
         assert req_body["new_cluster"] == cluster_spec
+        assert req_body["libraries"] == [
+            {"pypi": {"package": "mlflow<=%s" % mlflow.__version__}},
+        ]
 
 
 @pytest.mark.usefixtures("before_run_validations_mock", "runs_cancel_mock",
                          "dbfs_mocks", "cluster_spec_mock", "set_tag_mock")
+@pytest.mark.parametrize('libraries', [[{"pypi": {"package": "tensorflow"}}], None])
```

Comment:
Using `pytest.mark.parametrize` to dedup some duplicate test code here (combine tests with & without libraries into a single parametrized test)